src/map/cMap.cpp

```diff
@@ -188,12 +196,12 @@ bool cMap::canDeployUnitTypeAtCell(int iCell, int iUnitType)
 
     if (isWorm) {
         int wormId = getCellIdWormsLayer(iCell);
-        return global_map.isCellPassableForWorm(iCell) && wormId < 0;
+        return isCellPassableForWorm(iCell) && wormId < 0;
```

Comment:
Good find 👍  - no need to rely on `global_map` if we can access internal information.